### ISSUE TITLE
Mesh function in 3D scatter Plot

### ISSUE THREAD

**Issue by mehmetmuejde** [neuhausi/canvasXpress#52]
Connect the points in the 3D scatter plot as triple with lines and so span a plane.

Example: http://pgfplots.net/tikz/examples/mesh-plot/

**Comment by neuhausi** [comment]
Do you mean a lattice? Like in the loess fit

https://canvasxpress.org/html/scatter3d-3.html

![cx-scatter3d3](https://user-images.githubusercontent.com/254923/37440154-e8d29fc6-27d1-11e8-8097-5d18cf2bd8e8.png)

I can add it in version 18.9. I need to publish version 18.8 tonight to fix some bugs in the R interface.

Isaac

**Comment by mmuejde** [comment]
Yeah, it's like a "grid with triangular connections" between the data points. Your example is a regression. When you can add it in version 18.9, it would be great.

**Comment by neuhausi** [comment]
Cn we close this issue?

**Comment by neuhausi** [comment]
Not needed anymore
